### ISSUE TITLE
Cannot connect to Spa from Home Assistant ⩾ 2023.5.0

### ISSUE THREAD

**Issue by youtoofan** [mathieu-mp/homeassistant-intex-spa#47]
With the latest update of home assistant (2023.5.0), i lost the connection with my spa.
This is the log entry:

```
Logger: homeassistant.config_entries
Source: custom_components/intex_spa/__init__.py:48
Integration: Intex Spa ([documentation](https://github.com/mathieu-mp/homeassistant-intex-spa), [issues](https://github.com/mathieu-mp/homeassistant-intex-spa/issues))
First occurred: 12:42:01 (1 occurrences)
Last logged: 12:42:01

Error setting up entry Spa for intex_spa
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 387, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/config/custom_components/intex_spa/__init__.py", line 48, in async_setup_entry
    hass.config_entries.async_setup_platforms(entry, PLATFORMS)
AttributeError: 'ConfigEntries' object has no attribute 'async_setup_platforms'

```

**Comment by mathieu-mp** [comment]
Thank you for opening this issue.
I will fix it by the end of the week.

**Comment by Brectan** [comment]
Logger: homeassistant.config_entries
Source: custom_components/intex_spa/__init__.py:48
Integration: Intex Spa (documentation, issues)
First occurred: 17:48:16 (3 occurrences)
Last logged: 17:51:07

Error setting up entry Spa for intex_spa
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 387, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/config/custom_components/intex_spa/__init__.py", line 48, in async_setup_entry
    hass.config_entries.async_setup_platforms(entry, PLATFORMS)
AttributeError: 'ConfigEntries' object has no attribute 'async_setup_platforms'

**Comment by johanveld** [comment]
I set up my spa after winter. And am unable to connect to home assistant. I think it's the same issue.

**Comment by johanveld** [comment]
Can confirm above solution it works

**Comment by mathieu-mp** [comment]
Fixed: Please update to intex_spa ⩾ 0.2.2
